src/approaches/tamp_approach.py

```diff
@@ -28,6 +28,8 @@ def __init__(self, simulator: Callable[[State, Action], State],
         self._option_model = create_option_model(CFG.option_model_name,
                                                  self._simulator)
         self._num_calls = 0
+        self._metrics[
```

Comment:
from my poking around, it seems that `self._metrics` aggregates metrics for the entire approach (so over all the tasks) whereas `metrics` is the result from planning one task. `self._metrics` is initialized in the `BaseApproach` class

Reply:
oh right, that sounds good! in that case, is it correct to have this line be here? this would basically mean that min_num_skeletons_optimized and max_num_skeletons_optimized are calculated over all the tasks, is that what you want?

Reply:
yup